test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsFileParsingTests/SettingSectionTests.cs

```diff
@@ -409,5 +409,85 @@ public void SettingSection_Merge_WithTwoDifferentSections_Throws()
             ex.Should().BeOfType<ArgumentException>();
             ex.Message.Should().Be("Cannot merge two different sections.");
         }
+
+        [Fact]
+        public void SettingSection_Equals_WithSameElementName_ReturnsTrue()
+        {
+            var section1 = new VirtualSettingSection("section", new AddItem("key1", "val"));
+            var section2 = new VirtualSettingSection("section");
+
+            section1.Equals(section2).Should().BeTrue();
+        }
+
+        [Fact]
+        public void SettingSection_Equals_WithDifferentElementName_ReturnsFalse()
+        {
+            var section1 = new VirtualSettingSection("section", new AddItem("key1", "val"));
+            var section2 = new VirtualSettingSection("section1", new AddItem("key1", "val"));
+
+            section1.Equals(section2).Should().BeFalse();
+        }
+
+        [Fact]
+        public void VirtualSection_ElementName_IsCorrect()
+        {
+            var settingSection = new VirtualSettingSection("section", new AddItem("key1", "val"));
+
+            settingSection.ElementName.Should().Be("section");
+        }
+
+        [Fact]
+        public void ParsedSection_ElementName_IsCorrect()
+        {
+            var settingSection = new ParsedSettingSection("section", new AddItem("key1", "val"));
+
+            settingSection.ElementName.Should().Be("section");
+        }
+
+        [Fact]
+        public void AddItem_Clone_CopiesTheSameItem()
```

Comment:
Shouldn't this be `SettingSection_Clone_...`?  I only see `SettingSection.Clone()` called [below](https://github.com/NuGet/NuGet.Client/pull/2481/files#diff-cb95d5024ef689eeb0a76ac3e50cba9cR471) not `AddItem.Clone()`.  This test is directly testing `SettingSection.Clone()`.  The `<add />` element could be replaced with any setting element.  This file is for `SettingSection` tests only.

Reply:
This was an error from copying the test name.... Good catch!